### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by GQAdonis** [OpenFunction/OpenFunction#517] [General]
I noticed that nothing has been released in 5 months, and there seems to be very little activity.  Is this project still viable, or is there a replacement?

**Comment by shabbir-saifee-cmm** [comment]
Any updates?? Not activity in Discord or Slack either

**Comment by joebowbeer** [comment]
Relates https://github.com/OpenFunction/OpenFunction/issues/522

What are people using instead?
